chia/_tests/pools/test_pool_cmdline.py

```diff
@@ -1,20 +1,1050 @@
 from __future__ import annotations
```

Comment:
It's also important I think to add tests in this fashion: https://github.com/Chia-Network/chia-blockchain/blob/main/chia/_tests/wallet/test_signer_protocol.py#L690 which check that the options are parsed correctly into class instances.

Reply:
Added